brooklyn-tosca-transformer/src/main/java/org/apache/brooklyn/tosca/a4c/brooklyn/ToscaNodeToEntityConverter.java

```diff
@@ -349,29 +349,68 @@ private void configureConfigKeysSpec(EntitySpec spec, ConfigBag bag){
         return propertyMap;
     }
 
-
-    protected Map<String, Operation> getInterfaceOperations() {
-        final Map<String, Operation> operations = MutableMap.of();
+    protected Map<String, Operation> getStandardInterfaceOperations() {
+        Map<String, Operation> operations = MutableMap.of();
 
         if (indexedNodeTemplate.getInterfaces() != null) {
-            final ImmutableList<String> validInterfaceNames = ImmutableList.of("tosca.interfaces.node.lifecycle.Standard", "Standard", "standard");
-            final MutableMap<String, Interface> interfaces = MutableMap.copyOf(indexedNodeTemplate.getInterfaces());
-
-            for (String validInterfaceName : validInterfaceNames) {
-                Interface validInterface = interfaces.remove(validInterfaceName);
-                if (validInterface != null) {
-                    operations.putAll(validInterface.getOperations());
-                    if (!interfaces.isEmpty()) {
-                        log.warn("Could not translate some interfaces for " + this.nodeId + ": " + interfaces.keySet());
+            final ImmutableList<String> validInterfaceNames =
```

Comment:
this looks like it should be a constant, ie `private static final List<String> STANDARD_INTERFACE_KEYS`, which could be shared, rather than defined inline here